components/layout_thread/lib.rs

```diff
@@ -109,7 +109,7 @@ use style::traversal_flags::TraversalFlags;
 use style_traits::CSSPixel;
 use style_traits::DevicePixel;
 use style_traits::SpeculativePainter;
-use webrender_api::{units, ColorF, HitTestFlags, ScrollClamping};
+use webrender_api::{units, ColorF, HitTestFlags /*, HitTestFlags*/};
```

Comment:
Can the commented item be removed?

Reply:
Thanks for pointing this out. I have removed the commented out use statement.